lib/screens/login.dart

```diff
@@ -1,11 +1,14 @@
 import 'dart:developer';
+
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:go_router/go_router.dart';
 
-import 'package:squadquest/common.dart';
-import 'package:squadquest/app_scaffold.dart';
-import 'package:squadquest/controllers/auth.dart';
+import '../app_scaffold.dart';
+import '../common.dart';
+import '../components/phone_number_field.dart';
+import '../controllers/auth.dart';
+import '../services/supabase.dart';
```

Comment:
@IguJl15 is there a reason for this change or did your editor do it?

I'm going to merge and then adjust but I'm curious

Reply:
I have a [VSCode extension](https://marketplace.visualstudio.com/items?itemName=luanpotter.dart-import) to organize dart imports. They are organized according to the [Dart Guidelines](https://dart.dev/effective-dart/usage#prefer-relative-import-paths)'s "PREFER relative import paths".

I have a habit of just spamming the organize imports command and I haven't seen you organize imports that way. Let me know if you prefer to keep the current form (using `package:` denomination) and I will disable the extension for the repository.

Reply:
hmm the guidelines don't seem to recommend against using the package paths, I found them nice while I was refactoring a lot and VSCode kept screwing up automatically updating paths. It's also been helping me copy paste around more. I'd be down to do a bulk adjustment all in one PR at some point if it's the better pattern, but let's disable the extension in the repo's `.vscode/settings.json` for now so it's off for everyone so it's not happening file-by-file accidentally and muddying up PRs.